packages/card-validator/index.ts

```diff
@@ -107,16 +107,23 @@ export function validateCVC(
   cvc: string,
   cardNumber: string
 ): CardCVCValidationResult {
-  const validatedCard = validateNumber(cardNumber);
-  if (!validatedCard.isValid) {
+  // Check if the CVC only contains numbers with 3 or 4 digits
```

Comment:
I don't think we should be doing this because it will then always allow 3-4 digits even when we know the brand (which is the majority of cases). I'd probably make cardNumber optional and fallback on the regex when cardNumber is null/undefined?